src/Libs/OwlCore.WinUI/Threading/WeakEventListner.cs

```diff
@@ -53,7 +53,7 @@ public WeakEventListener(TInstance instance)
         /// <param name="eventArgs">Event arguments.</param>
         public void OnEvent(TSource source, TEventArgs eventArgs)
         {
-            var target = (TInstance)_weakInstance.Target;
+            var target = _weakInstance.Target as TInstance;
```

Comment:
`TInstance`? isn't too general name?

Reply:
The suggestion doesn't seem related to the changes made for this PR. This code is copied from the Community Toolkit, so we won't be changing it unless absolutely needed.